Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -105,6 +105,8 @@ public class VideoRenderer: RTCMTLVideoView {
         self.track?.trackId
     }
     
+    private var viewSize: CGSize?
```

Comment:
```suggestion
    private var trackSize: CGSize = .zero
    private lazy var scale: CGFloat = UIScreen.main.scale
```

Reply:
It's actually the view size, not the track size, so the naming is correct. Good point about the scale, will move it here.
Btw, should be optional, because we have a default track size - we should change it only when we know the view size.

Reply:
done